LICENSE

```diff
@@ -1,51 +1,23 @@
-MICROSOFT SOFTWARE LICENSE TERMS
+The MIT License (MIT)
 
-MICROSOFT INSTRUMENTATION ENGINE SOFTWARE DEVELOPMENT KIT
+Copyright (c) .NET Foundation and Contributors
```

Comment:
Why .NET foundation? Shouldn't this be Microsoft?

Reply:
Correct, thanks for the catch!